apps/web/src/pages/settings/tabs/ApiKeysCard.cy.tsx

```diff
@@ -79,12 +79,17 @@ describe('Input Component', () => {
 
   it('should copy Api Key and Application Identifier to the clipboard', () => {
     cy.wait('@getApiKeys');
+    cy.window().then((win) => {
+      cy.spy(win.navigator.clipboard, 'writeText').as('copy');
+
+      return;
+    });
     cy.get("[data-test-id='api-key-tooltip']").click();
-    cy.window().its('navigator.clipboard').invoke('readText').should('equal', apiKeys[0].key);
```

Comment:
I changed the way to test the clipboard functionality.
I found that the clipboard is available only if permission is granted and `readText` seems to not work expectedly in cypress.